README.md

```diff
@@ -188,6 +220,18 @@ token [here](https://app.snyk.io/login?utm_campaign=Code-Ready-Analytics-2020&ut
   }
   ```
 
+  For example, setting `exclude` attribute in the `go.mod` file:
+  ```
+  exclude golang.org/x/sys v1.6.7
+
+  exclude (
+      golang.org/x/sys v1.6.7
+  )
+  ```
+
+  For example, creating an alternative file to `requirements.txt`, like `requirements-dev.txt`
+  or `requirements-test.txt` and adding the dev or test dependencies there instead.
```

Comment:
Suggest a rewrite of this sentence (lines 232 & 233).  For example:

``You can create an alternative file to `requirements.txt`, for example, a `requirements-dev.txt` or a `requirements-test.txt` file where you can add the development or test dependencies there.``